### ISSUE TITLE
Rating labels have low contrast

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu/app-center#1746]
<img width="1050" alt="grafik" src="https://github.com/user-attachments/assets/e826b658-3f42-46fd-88bb-b9458bce1885">

The rating labels do not have enough contrast. I think some custom green is being used instead of yaru's success color

**Comment by anasereijo** [comment]
@Feichtmeier good catch, it doesn't have much contrast indeed! 

I suppose that the yaru success colour is accessible with the dark theme? If so, happy to go with that! Thank you

**Comment by spydon** [comment]
@anasereijo this is solved here: https://github.com/ubuntu/app-center/pull/1765
but it can't be merged until the fix the CLA check.

**Comment by spydon** [comment]
That one isn't using the correct yaru colors though, I'll fix it up
